### PR TITLE
My wish for today is to fix the worst line of documentation in the pr…

### PR DESCRIPTION
**Pull request by yb66**
…oject.

14 was the only number to avoid.

Regards,
iain

### CODE REVIEW

File: cont.c [hunk 1]
```diff
@@ -1411,13 +1411,13 @@ rb_cont_call(int argc, VALUE *argv, VALUE contval)
  *    end
  *
  *    puts fiber.resume 10
- *    puts fiber.resume 14
- *    puts fiber.resume 18
+ *    puts fiber.resume 1_000_000
```
Comment (mikegee) [inline cont.c]: Consider something like this to clarify that no math will be occurring on the second call:

```suggestion
 *    puts fiber.resume :second_return_value
```

**Comment by mikegee** [comment]
I appreciate the content of this pull request, but the title and description could be improved to clarify your intentions and how this change addresses the problem.

**Comment by yb66** [comment]
Hi @mikegee,

Apologies and fair enough, I was probably a tad frustrated and fell back on standard British sarcasm too readily.

Documentation is for those who don't know, remember, or understand (to any degree) the language, it should attempt to be clear above all other things. The example given is needlessly unclear because if you use a block it's common for arguments to be resused on every entry to the block. In Fiber's case this is not so.

First time round 10 goes in, 12 comes out.  
Second time round 14 goes in, 14 comes out… was that because 14 is 12 + 2 or because it's "the return value of the call to Fiber.yield". It's the latter because it says so but why does the example need to make anyone think the former?

Using different numbers makes it immediately clear what's happening whether the description is there or not.

Regards,
iain

**Comment by ioquatix** [comment]
You can make it 42 and then it is even more funny.

**Comment by yb66** [comment]
@ioquatix Considering I was looking at the documentation because I was reading your [article about fibers](https://www.codeotaku.com/journal/2018-11/fibers-are-the-right-solution/index) it's already irony laden enough :)

How about…?

```
class FiberError
  def exception(message="The Vogons blew up your planet")  
    RuntimeError.new message    
  end      
end 


fb = Fiber.new do |any_number|
  second = Fiber.yield any_number.to_s 13    
end 

fb.resume 6 * 9
# "42"
fb.resume 6 * 9
# 54
fb.resume 6 * 9
RuntimeError: The Vogons blew up your planet
from (pry):47:in `resume'
```

**Pull request merged**